### ISSUE TITLE
Slack Link Dead

### ISSUE THREAD

**Issue by pascal-mueller** [hapijs/hapi#4157]
The Slack Link on the Homepage https://hapi.dev/support/ is dead.

**Comment by devinivy** [comment]
Good call!  Thanks for reporting.  For those with the same issue, here's the good link: https://join.slack.com/t/hapihour/shared_invite/zt-g5ortpsk-ErlnRA2rUcPIWES21oXBOg.

An aside— I am moving this issue to the hapi.dev repository.
